src/jet/elements/environment.rs

```diff
@@ -53,21 +53,21 @@ pub struct ElementsUtxo {
 // an Arc: we'd have a lifetime parameter <'a> that would cause us trouble.
 pub struct ElementsEnv {
     /// The CTxEnv struct
-    pub(super) c_tx_env: CElementsTxEnv,
+    pub(crate) c_tx_env: CElementsTxEnv,
     /// The elements transaction
-    pub(super) tx: Arc<elements::Transaction>,
+    pub(crate) tx: Arc<elements::Transaction>,
     /// The input utxo information corresponding to outpoint being spent.
-    pub(super) utxos: Vec<ElementsUtxo>,
+    pub(crate) utxos: Vec<ElementsUtxo>,
     /// the current index of the input
-    pub(super) ix: u32,
+    pub(crate) ix: u32,
     /// Commitment merkle root of the script being executed
-    pub(super) script_cmr: Cmr,
+    pub(crate) script_cmr: Cmr,
     /// Control block used to spend this leaf script
-    pub(super) control_block: ControlBlock,
+    pub(crate) control_block: ControlBlock,
     /// Optional Annex.
-    pub(super) annex: Option<Vec<u8>>,
+    pub(crate) annex: Option<Vec<u8>>,
     /// Genesis block hash
-    pub(super) genesis_hash: BlockHash,
+    pub(crate) genesis_hash: BlockHash,
```

Comment:
I think these should be pub(super). And have pub accessor methods to access these elsewhere in the codebase. I don't think others should be setting anything in ElementsEnv. Once, it is created it should not mutated.

Reply:
I see. That is the difference between member visibility and getters. Got it.